maven-setup/src/main.ts

```diff
@@ -23,19 +24,35 @@ export async function run(): Promise<void> {
         const accessToken = await auth.getAccessToken(input.getDevelocityAccessKey(), input.getDevelocityTokenExpiry())
 
         const downloadFolder = maven.mavenBuildTool.getBuildScanWorkDir()
-        let develocityMavenExtensionJar = ''
-        let ccudMavenExtensionJar = ''
+        let develocityMavenExtensionMavenOpts = ''
         if (input.getDevelocityInjectionEnabled() && input.getDevelocityUrl()) {
```

Comment:
Could we extract this logic to a dedicated method to leave the run method as simple as possible?

Reply:
Extracted